### PR TITLE
Add feature spec for creating a collection

### LINKED ISSUE #7: Create a test for creating a collection
This happens in argo and uses the registration service in dor-services-app

### PR DESCRIPTION
**Pull request by mjgiarlo**
Fixes #7 

Includes:
* Rejigger Capybara to allow multiple specs to run serially using the initial set of credentials.
* Extract authentication code to a spec helper module
* Remove debugging statement printing download path
* Rename preassembly feature spec to clarify its intention, and to signal that going forward we expect each feature spec to test a single feature.

Note that I ran the tests and can attest that they passed for me. It might be useful to have one other @sul-dlss/infrastructure-team member pull down this branch and verify that the specs pass for them as well.

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -28,11 +27,12 @@
 #   end
 # end
 
-Capybara.current_driver = :my_firefox_driver
+Capybara.default_driver = :my_firefox_driver
```
Comment (mjgiarlo) [inline spec/spec_helper.rb]: This change was required to allow multiple specs to use the same browser session.

File: spec/support/authentication_helpers.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      print "SUNet ID: "
+      username = $stdin.gets
+      username.strip
+    end
+
+    @@password ||= begin
+      print "Password: "
+      password = $stdin.noecho(&:gets)
+      # So the user knows we're off the password prompt
+      puts
```
Comment (mjgiarlo) [inline spec/support/authentication_helpers.rb]: I know it looks weird to have the puts here, but if it's the last line in the begin/end block, it interferes with the return value. And if it's outside the block, there's unnecessary newlines in the spec output.

**Comment by peetucket** [comment]
Pulled the branch and ran tests and it works for me.

File: spec/features/create_collection_spec.rb [hunk 3]
```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+require 'io/console'
+require 'random_word'
+
+RSpec.describe 'Create a collection in Argo', type: :feature do
```
[the remark this answers is not in the document]
Comment (mjgiarlo) [reply spec/features/create_collection_spec.rb]: Made the change. thanks!

**Pull request merged**